libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -4072,6 +4072,27 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                 new ExpressionEvaluator(ExpressionType.String, Apply(args => JsonConvert.SerializeObject(args[0]).TrimStart('"').TrimEnd('"')), ReturnType.String, ValidateUnary),
                 Comparison(ExpressionType.Bool, args => IsLogicTrue(args[0]), ValidateUnary),
                 new ExpressionEvaluator(ExpressionType.Xml, ApplyWithError(args => ToXml(args[0])), ReturnType.String, ValidateUnary),
+                new ExpressionEvaluator(
+                    ExpressionType.FormatNumber, 
+                    ApplyWithError(
+                        args => 
+                        {
+                            double? result = null;
+                            string error = null;
+                            var originalNum = CultureInvariantDoubleConvert(args[0]);
+                            if (args[1] is int digits)
```

Comment:
use "IsInteger" and "Convert.ToInt32()" instead to support uint/short/ushort/long data type